cardform/src/main/java/com/mercadolibre/android/cardform/presentation/ui/formentry/CardNameFragment.kt

```diff
@@ -53,10 +54,12 @@ internal class CardNameFragment : InputFragment() {
     override fun bindViewModel() {
         with(viewModel) {
             nameLiveData.nonNullObserve(viewLifecycleOwner) { data ->
-                nameCardEditText.configure(data) {
-                    updateInputData(CardFilledData.Name(it))
-                    isInputValid = nameCardEditText.validate()
-                    nameCardEditText.clearError()
+                nameCardEditText?.configure(data) {
+                    nameCardEditText?.apply {
```

Comment:
medio raro tener que chequear por null dos veces no? haria el apply solo y el configure sobre el it o algo asi

Reply:
Dale!